hudi-utilities/src/test/java/org/apache/hudi/utilities/testutils/UtilitiesTestBase.java

```diff
@@ -364,5 +385,32 @@ public static String toJsonString(HoodieRecord hr) {
     public static String[] jsonifyRecords(List<HoodieRecord> records) {
       return records.stream().map(Helpers::toJsonString).toArray(String[]::new);
     }
+
+    public static void addAvroRecord(
```

Comment:
does this need to be public? can we switch to private.

Reply:
Sure thing. changed.